Source/Serilog.Enrichers.Span/ActivityEnricher.cs

```diff
@@ -33,15 +33,25 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
                 AddSpanId(logEvent, activity);
                 AddTraceId(logEvent, activity);
                 AddParentId(logEvent, activity);
+                AddTags(logEvent, activity);
 #else
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(SpanId, new ScalarValue(activity.GetSpanId())));
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(TraceId, new ScalarValue(activity.GetTraceId())));
                 logEvent.AddPropertyIfAbsent(new LogEventProperty(ParentId, new ScalarValue(activity.GetParentId())));
 #endif
             }
         }
+
 #if NET5_0_OR_GREATER
 
+        private static void AddTags(LogEvent logEvent, Activity activity)
+        {
+            foreach (var item in activity.Tags)
+            {
+                logEvent.AddPropertyIfAbsent(new LogEventProperty(item.Key, new ScalarValue(item.Value)));
```

Comment:
There is a chance of collisions here, since we are adding these properties to the root of the log event. Perhaps we can group all tags in the log event together into a tags object.